opengrok-indexer/src/test/java/org/opengrok/indexer/analysis/AnalyzerFrameworkTest.java

```diff
@@ -0,0 +1,12 @@
+package org.opengrok.indexer.analysis;
```

Comment:
I don't think it's a good idea to keep this here. Either add some tests or remove it -> YAGNI.

Reply:
I was about to fill it but it looks like it won’t happen anywhen soon so I’ll delete for now

Reply:
I filled the test with actual code